util-core/src/test/scala/com/twitter/util/CancellableTest.scala

```diff
@@ -3,6 +3,39 @@ package com.twitter.util
 import org.scalatest.WordSpec
 
 class CancellableTest extends WordSpec {
+  "Cancellable" should {
+    "test variable isCancelled" in {
+      assert(!Cancellable.nil.isCancelled)
+    }
```

Comment:
This is a test on the `Cancellable` companion object's `nil` property and not the `Cancellable` interface, correct? Should we give it a clearer test title (i.e. `"test Cancellable.nil is always cancelled"`)?

Reply:
Okay!